### DISCUSSION TITLE
(not a) Steam problem

### DISCUSSION THREAD

**Discussion by 0NDRS** [Lamer87/Space_Engineers_DLC_unlocker#48] [Q&A] [answered]
After runningunlocker by tutorial it accessed my steam account and started sending scam steam giftcard link to all of my friends, by logging history i can see that he is logging from Russia.  Do anyone know something about this????

**Comment by Lamer87** [comment, marked as answer]
You can easily view the file in your browser by clicking on this link: [CreamAPI_Tool.cmd](https://github.com/Lamer87/Space_Engineers_DLC_unlocker/blob/main/CreamAPI_Tool.cmd)

As you can see, all this file does is move/delete/rename files.

That said, you probably got some virus from somewhere else, assuming what you say is true and not completely made up ;)

I'll leave this thread open so that if others have had the same experience they can come here to confirm,
as well as prove that there is nothing to hide.
